packages/cli/src/utils/loadConfig.test.ts

```diff
@@ -0,0 +1,38 @@
+import path from 'node:path'
+
+import { jestContext } from '@prisma/get-platform'
+import { HelpError } from '@prisma/internals'
+
+import { loadConfig } from './loadConfig'
+
+const ctx = jestContext.new().assemble()
+
+describe('loadConfig', () => {
+  it('loads config from file', async () => {
+    ctx.fixture('prisma-config')
+
+    const config = await loadConfig('./prisma.config.ts')
+
+    expect(config).toMatchObject({
+      experimental: true,
+      loadedFromFile: path.join(ctx.fs.cwd(), './prisma.config.ts'),
```

Comment:
When passing files to path where we specify a basedir explicitly, we should not use the `./` prefix in strings.